xdmod-run-ingestor.sh

```diff
@@ -2,4 +2,5 @@
 
 cp /mnt/xdmod_init/xdmod_init.json /etc/xdmod/xdmod_init.json
 /usr/bin/xdmod-get-config-files 
+/usr/bin/sleep 30
```

Comment:
@rob-baron Was this related to the database connection dropping issue?

Reply:
@larsks This was thrown in as an attempt to work around the database connection dropping issue.  

2 things:

1) I am not certain if this works
2) This cannot be here long term as it will not meet the overall performance requirements

I have kept it here as I am developing this as it was useful for development.